yb-voyager/src/query/queryissue/constants.go

```diff
@@ -51,6 +51,8 @@ const (
 
 	AGGREGATE_FUNCTION             = "AGGREGATE_FUNCTION"
 	AGGREGATION_FUNCTIONS_NAME     = "Aggregate Functions"
```

Comment:
what all cases this AGGREGATE FUNCTION type is used?
any specific reason for not name it like `RANGE AGGREGATE...`, current one doesn't seems to be highlighting the issue in case of schema DDL(classsified as Unsupported Features) have this.

Reply:
it has only three functions as of now And I don't feel the need to separate the issues based on type of aggregate function we will anyways have to work on reporting the actual function name in the query. I already added them in the `Details` map of issue and whenever we pick that up it will show what is the actual issue.